### ISSUE TITLE
REACT: Deletion of constructs not always working?

### ISSUE THREAD

**Issue by HayleyMills** [CLOSER-Cohorts/archivist#633]
I do not know the extent of the problem, but I have an example of some statements which were deleted, but they remained in the database and in the summary pages. 
Document view with no s_outro_i https://closer-archivist.herokuapp.com/instruments/sws_covid_pq1/
Construct view with no s_outro_i https://closer-archivist.herokuapp.com/instruments/sws_covid_pq1/build/constructs/
Statement summary with s_outro_i https://closer-archivist.herokuapp.com/instruments/sws_covid_pq1/build/ccStatements

I couldn't see any issues with the database (they have the correct parents/position) so I think it might be a react issue? I tried clearing the cache but that didn't do anything. 

These deleted items are exported in the XML and are imported in the same way e.g. https://closer-archivist-staging.herokuapp.com/instruments/sws_covid_pq1/ 

I'm not sure if there is an easy way to tell if this has happended in other instruments?
I have tested deleting statements in the staging and live versions now and there doesn't seem to be a problem.

**Comment by simonreed** [comment]
CcSequence "15694" with label "Open Ended"  contained these statements. Somehow it was placed as a child of the top level sequence for that instrument. We only show the first child of the top sequence so it wasn't shown. 

I can't see how that was done. I've now moved it back into the position it would have originally been in so that it's visible here https://closer-archivist.herokuapp.com/instruments/sws_covid_pq1/ and now deleted it using the UI. That has now worked as expected.

Unclear how "Open Ended" was moved into part of the tree that wasn't visible but I can't replicate. Deleting a sequence, now deletes the construct children so working as expected.
